res/layout/edit_box_dialog.xml

```diff
@@ -18,19 +18,17 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:gravity="clip_horizontal"
-    android:orientation="vertical" >
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:gravity="clip_horizontal"
+              android:orientation="vertical">
```

Comment:
@AndyScherzinger @tobiasKaminsky Do you want to me to remove formatting and only make required changes?

If that is the case then do I revert the commit, make changes and push again. Right?

Sorry for taking up so much time. I hope I can improve in this area.